fs/accounting/stats_groups.go

```diff
@@ -349,3 +378,26 @@ func (sg *statsGroups) reset() {
 	sg.m = make(map[string]*StatsInfo)
 	sg.order = nil
 }
+
+// delete removes all references to the group.
+func (sg *statsGroups) delete(group string) {
+	sg.mu.Lock()
+	defer sg.mu.Unlock()
+	stats := sg.m[group]
```

Comment:
How about `stats, ok := sg.m[group]`?

Reply:
If key is not found we get zero value so it's not needed. I've modified `get` too to keep it consistent.